### DISCUSSION TITLE
I Haven't got my github education pack (its been 5 days since approval)

### DISCUSSION THREAD

**Discussion by giabaoo420** [community/community#144937] [GitHub Education]
### Hi everyone,

![Capture](https://github.com/user-attachments/assets/709b9b12-670c-4a61-9ef7-9410492e6894)

**Comment by davevad93** [comment]
Hi @giabaoo420 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @giabaoo420 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
